astropy/units/si.py

```diff
@@ -109,6 +107,8 @@
     doc="arc second: angular measurement",
 )
 # These special formats should only be used for the non-prefix versions
+deg._format = {"latex": r"{}^{\circ}", "unicode": "°"}
+arcmin._format = {"latex": r"{}^{\prime}", "unicode": "′"}
 arcsec._format = {"latex": r"{}^{\prime\prime}", "unicode": "″"}
```

Comment:
Having a `._format` additionally to the `.format` for a unit seems a bit hacky to me, and is not straightforward to understand. Sure this was invented long time ago, but shouldn't the attribute name changed here to something more descriptive before it gets extended? Like `.format_simple` or so`

Reply:
Definitely hacky, but clearly with precedent... Ideally, we have some kind of hint in `def_unit` that the format should only be used for non-prefix, and for power=1. But maybe best to keep that for another PR...

Reply:
Agreed. A refactor / simplification would be good for a followup, but for this PR extending an existing framework is a good solution.

Reply:
OK, agreed